src/wp-includes/functions.php

```diff
@@ -1793,7 +1793,7 @@ function is_blog_installed() {
 			continue;
 		}
 
-		// One or more tables exist. This is not good.
+		// One or more tables do not exist. This is not good.
```

Comment:
I'm not sure this is correct.

- All `true` returns are earlier in the function.
- The code just above this inline comment checks if table does not exist, or if it exists but it's empty. In this case, the loop can be skipped. Otherwise, it should add an error because the function is about to return `false` when there are existing, populated tables in the database.

Reply:
reverted